src/components/panel/editForm/Panel.edit.display.js

```diff
@@ -95,6 +95,13 @@ export default [
           json: { '!==': [{ var: 'data.breadcrumb' }, 'none'] }
         }
       },
+      {
+        input: true,
```

Comment:
This needs a tooltip. Also, it should be conditional and only show up if they have breadcrumbClickable set to false.